force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,7 +29,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        String questionOne; 
+        if (x > y) {
+            questionOne = 'Hello World!';
+        }
+        else if (x == null || y == null) {
```

Comment:
Since you are setting your variable to null in both else blocks, you only need the one. I would advise ruling out a null value first. The shortened way to do it would be something like this:

```
if (x != null && y != null && x > y) {
   return 'Hello World!'
} 
   return null;
```